src/Object/ObjectFactory.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the zenstruck/foundry package.
+ *
+ * (c) Kevin Bond <kevinbond@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Zenstruck\Foundry\Object;
```

Comment:
In foundry-next, I have this in `Zenstruck\Foundry`. I don't have a hard opinion on this I just didn't like the duplicated _Object_ word in the namespace and this will always be part of foundry core (once we make the split packages)

Reply:
fixed ✅